keyboards/undead60m/readme.md

```diff
@@ -0,0 +1,22 @@
+# Undead 60M
```

Comment:
Please follow the normal standard readme template? 

https://github.com/qmk/qmk_firmware/blob/fab1af415413e14c04c41dc3716c75500d67baf3/quantum/template/avr/readme.md#L3-L20